Source/Cvent.SchemaToPoco.Core/Util/StringUtils.cs

```diff
@@ -19,7 +19,17 @@ public static class StringUtils
             'D',
             's',
             'S'
-        };
+        };
+
+        /// <summary>
+        ///     lowercase the first letter in a string.
+        /// </summary>
+        /// <param name="value">The string.</param>
+        /// <returns>A first letter lowercased string.</returns>
+        public static string LowerFirst(string value)
+        {
+            return Char.ToLowerInvariant(value[0]) + value.Substring(1);

```

Comment:
You should probably check for null/empty and cut out early before doing this (see other methods below this for capitalizing, etc.